tests/core/management/test_event_router.py

```diff
@@ -137,8 +140,13 @@ def test_event_router_output_components(event_router: EventRouter):
     assert {"Out1", "Out2", "Mid1"} == event_router.output_components
 
 
+def test_event_router_isolated_components(event_router: EventRouter):
```

Comment:
Add test for graphs with:

2 isolated graphs of 2 components
2 isolated graphs of 1 component
3 isolated graphs of {3, 1, 1} components
3 isolated graphs of {3, 2, 2} components
1 isolated graph of 1 component

Is 0 graphs (0 components) a valid arrangement that isn't already under test?